iocore/net/P_UnixNet.h

```diff
@@ -90,7 +90,7 @@ struct EventIO {
   bool syscall         = true;    ///< if false, disable all functionality (for QUIC)
   int type             = 0;       ///< class identifier of union data.
   union {
-    Continuation *c;
```

Comment:
This was only used to do the ratchet conversion from pointer to NetAccept to Continuation.